src/main/java/cs361/battleships/models/Board.java

```diff
@@ -161,6 +169,27 @@ else if (capNumB == 0){
 					hitShip.getCCB().nohit();
 				}
 			}
+			else if (attackResult.getResult() == AtackStatus.HIT && hitShip.getCCM().getRow() == s.getRow() && hitShip.getCCM().getColumn() == s.getColumn() && hitShip.getKind().equals("SUB")){
+				if (capNumS == 1) {
+					Square tmpS = hitShip.getOccupiedSquares().get(1);
+					attacks.add(attackResult);
+					attackResult = hitShip.attack(tmpS.getRow(), tmpS.getColumn());
+					attacks.add(attackResult);
+					tmpS = hitShip.getOccupiedSquares().get(2);
+					attackResult = hitShip.attack(tmpS.getRow(), tmpS.getColumn());
+					attacks.add(attackResult);
+					tmpS = hitShip.getOccupiedSquares().get(3);
+					attackResult = hitShip.attack(tmpS.getRow(), tmpS.getColumn());
+					attacks.add(attackResult);
+					tmpS = hitShip.getOccupiedSquares().get(4);
+					attackResult = hitShip.attack(tmpS.getRow(), tmpS.getColumn());
+				}
+				else if (capNumS == 0){
+					capNumS++;
+					attackResult.setResult(AtackStatus.CAPHIT);
+					hitShip.getCCM().nohit();
+				}
+			}
```

Comment:
This follows what we did for the other types of ships.  Looks good and keeps the code consistent.